<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoViewer</class>
 <widget class="QWidget" name="PhotoViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>448</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>PhotoViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PhotoViewer
{
background-color: rgba(0, 0, 0,120);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#close
{
border:none;
border-image: url(:/res/photoview/gza.png)
}
QPushButton#close:hover
{
border:none;
border-image: url(:/res/photoview/fdt.png)
}
QPushButton#close:pressed
{
border:none;
border-image: url(:/res/photoview/fds.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="left">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#left
{
border:none;
border-image: url(:/res/talk/obg.png);
}
QPushButton#left:hover
{
border:none;
border-image: url(:/res/talk/orf.png)
}
QPushButton#left:pressed
{
border:none;
border-image: url(:/res/talk/ori.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="right">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#right
{
border:none;
border-image: url(:/res/talk/obh.png)
}
QPushButton#right:hover
{
border:none;
border-image:url(:/res/talk/org.png)
}
QPushButton#right:pressed
{
border:none;
border-image: url(:/res/talk/orh.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottom_layout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolBar" name="tool">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>340</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolBar#tool
{
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color: rgba(1, 1, 1,120);
}</string>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <addaction name="save"/>
       <addaction name="openByDefaultViewer"/>
       <addaction name="share"/>
       <addaction name="separator"/>
       <addaction name="zoomin"/>
       <addaction name="sizeShow"/>
       <addaction name="zoomout"/>
       <addaction name="realSize"/>
       <addaction name="separator"/>
       <addaction name="fullScreen"/>
       <addaction name="turnRight"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="save">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/ogv.png</normaloff>:/res/talk/ogv.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="openByDefaultViewer">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/ohr.png</normaloff>:/res/talk/ohr.png</iconset>
   </property>
   <property name="text">
    <string>view</string>
   </property>
   <property name="toolTip">
    <string>View Default Viewer</string>
   </property>
  </action>
  <action name="share">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/photoview/ooz.png</normaloff>:/res/photoview/ooz.png</iconset>
   </property>
   <property name="text">
    <string>share</string>
   </property>
   <property name="toolTip">
    <string>Share</string>
   </property>
  </action>
  <action name="zoomin">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/search.png</normaloff>:/res/talk/search.png</iconset>
   </property>
   <property name="text">
    <string>zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="zoomout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/search.png</normaloff>:/res/talk/search.png</iconset>
   </property>
   <property name="text">
    <string>zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="realSize">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/setting.png</normaloff>:/res/talk/setting.png</iconset>
   </property>
   <property name="text">
    <string>real size</string>
   </property>
   <property name="toolTip">
    <string>Show Real Size</string>
   </property>
  </action>
  <action name="fullScreen">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/ohr.png</normaloff>:/res/talk/ohr.png</iconset>
   </property>
   <property name="text">
    <string>look full screen</string>
   </property>
   <property name="toolTip">
    <string>Look Full Screen</string>
   </property>
  </action>
  <action name="turnRight">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/otb.png</normaloff>:/res/talk/otb.png</iconset>
   </property>
   <property name="text">
    <string>turn right</string>
   </property>
   <property name="toolTip">
    <string>Turn Right</string>
   </property>
  </action>
  <action name="sizeShow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="saveToCloud">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/nsm.png</normaloff>:/res/talk/nsm.png</iconset>
   </property>
   <property name="text">
    <string>saveToCloud</string>
   </property>
   <property name="toolTip">
    <string>Save To Cloud</string>
   </property>
  </action>
  <action name="saveToSpace">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/talk/ows.png</normaloff>:/res/talk/ows.png</iconset>
   </property>
   <property name="text">
    <string>saveToSpace</string>
   </property>
   <property name="toolTip">
    <string>Save To Space</string>
   </property>
  </action>
  <action name="ensherin">
   <property name="icon">
    <iconset>
     <normaloff>:/res/talk/qfav_misc_web_btn_favorite_pressed.png</normaloff>:/res/talk/qfav_misc_web_btn_favorite_pressed.png</iconset>
   </property>
   <property name="text">
    <string>ensherin</string>
   </property>
   <property name="toolTip">
    <string>Ensherin</string>
   </property>
  </action>
  <action name="sendToMobile">
   <property name="icon">
    <iconset>
     <normaloff>:/res/talk/fdi.png</normaloff>:/res/talk/fdi.png</iconset>
   </property>
   <property name="text">
    <string>sendToMobile</string>
   </property>
   <property name="toolTip">
    <string>Send To Mobile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>PhotoViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
